### PR TITLE
Faq

### LINKED ISSUE #915: Create an FAQ
Creating an issue to track the idea and to get input on what the FAQs are. I've started writing something locally. 

Current questions:

- What is decomp.me for?
- Do do you mean by 'match it'?
- What's a 'scratch'?
- What's 'context' for?
- How does decomp.me work?
- Where do I start?
- Can you add 'X' platform? e.g. PlayStation 3
- How do I report bugs?

### PR DESCRIPTION
**Pull request by mkst**
Creating a PR to gather feedback.

Closes #915

### CODE REVIEW

**Comment by marijnvdwerf** [comment]
(Terse because I’m on mobile. Nice job on adding the faq)

- Textual content should be left aligned instead of justified for easier reading
- Headings should probably have an id/name and way to get a `/faq#what-is-a-scratch` link
- Wouldn’t it be easier (editing-wise) to have the faq in a md file that gets turned into TS on compile time? (E.g https://nextjs.org/docs/app/building-your-application/configuring/mdx)
- Not sure if the page is long enough, but either a TOC or making sections collapsible could be a good way to help people get the answer they’re looking for
- Some info about the scoring algorithm?
- Maybe the longer paragraphs (good length btw!) could use some emphasis or links to make them a bit more visually engaging.

**Review by jpburnett — COMMENTED** [review]
Just some thoughts. Not sure if a "How do I make a scratch" would make sense? There is sections explaining that, but that could be more of a tutorial rather than FAQ?

**Comment by ethteck** [comment]
This is a great start that I think can be improved over time with some of the more QOL features like @marijnvdwerf mentioned. If no one has any objections, I think let's merge this soon

File: frontend/src/app/(navfooter)/faq/page.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        <p className="my-4">
+            This is something that you are able to do yourself.
+            The compilers used by decomp.me can be found in our <FaqLink href="https://github.com/decompme/compilers">compilers repository</FaqLink>.
+            Once the compiler has been added to that repo, it is very simple to add it to decomp.me, see <FaqLink href="https://github.com/decompme/decomp.me/pull/910">this PR</FaqLink> for an example.
+        </p>
+
+        <h2 className={subtitle}>Can you add 'X' platform, e.g. PlayStation 3?</h2>
+        <p className="my-4">
+            The platforms that decomp.me supports are driven by the support for those platforms in the underlying tools that make up decomp.me.
+            If these tools support the architecture for the new platform, and you have the compiler available, it is a straightforward process to add it to decomp.me.
```
[the remark this answers is not in the document]
Comment (nanaian) [reply frontend/src/app/(navfooter)/faq/page.tsx:113]: A contributing document that explains how to add a new compiler/platform (not preset - they're in the db now!) would be in order

[the remark this answers is not in the document]
Comment (ethteck) [reply frontend/src/app/(navfooter)/faq/page.tsx:113]: I agree, but I don't think it should hold up this PR. it's something that can be an issue and later created / linked to from this page

**Pull request merged**